go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/numaproj/numaflow-go
 go 1.20
 
 require (
+	github.com/fsnotify/fsnotify v1.6.0
```

Comment:
perhaps we should use go.mod for examples?

Reply:
+1

Reply:
done